src/views/v2/month/mobile-events/mobile-day/mobile-event/featured-image.php

```diff
@@ -22,13 +22,17 @@
 
 ?>
 <div class="tribe-events-calendar-month-mobile-events__mobile-event-featured-image-wrapper">
-	<div class="tribe-events-calendar-month-mobile-events__mobile-event-featured-image tribe-common-c-image tribe-common-c-image--bg">
-		<div
-			class="tribe-common-c-image__bg"
-			style="background-image: url('<?php echo esc_url( $event->thumbnail->full->url ); ?>');"
-			role="img"
-			aria-label="<?php echo esc_attr( get_the_title( $event->ID ) ); ?>"
-		>
-		</div>
-	</div>
+	<img
+		src="<?php echo esc_attr( $event->thumbnail->full->url ); ?>"
```

Comment:
:no_entry_sign: Wrong escaping function. href and src attributes should be escaped by `esc_url()`, not by `esc_attr()`.

<sub>hash: d77db59780eaaeef85d6fadf806b846d</sub>